Confus/Confus/Flag.cpp

```diff
@@ -109,71 +117,100 @@ namespace Confus {
         particleAffector->drop();
     }
 
-    irr::video::SColor Flag::getColor() {
+    irr::video::SColor Flag::getColor() 
+    {
         switch(*m_TeamIdentifier)
         {
-        case ETeamIdentifier::TEAM_BLUE:
+        case ETeamIdentifier::TeamBlue:
             return { 255, 0, 0, 255 };
-        case ETeamIdentifier::TEAM_RED:
+        case ETeamIdentifier::TeamRed:
             return { 255, 255, 0, 0 };
         default:
             return { 255, 255, 255, 255 };
         }
     }
 
 	//This class handles what to do on collision
-	void Flag::captureFlag(Player* a_PlayerObject) {
+	void Flag::captureFlag(Player* a_PlayerObject) 
+    {
 		//Somebody is already carrying the flag
-		if (*m_FlagStatus == EFlagEnum::FLAG_TAKEN)
+		if (*m_FlagStatus == EFlagEnum::FlagTaken)
```

Comment:
I think we have discussed these early returns before although it is already in, preferably it's not there. At the very least, do put curly brackets around it